appengine/webpack/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "appengine-webpack",
+  "description": "An example of using Webpack with Node.js on Google App Engine.",
+  "version": "0.0.1",
+  "private": true,
+  "license": "Apache Version 2.0",
+  "engines": {
+    "node": "~4.2"
+  },
+  "scripts": {
+    "bundle": "webpack --config webpack.config.js",
+    "prestart": "npm run bundle",
```

Comment:
Are we looking to explain this stuff in the readme, or moving it all to c.g.c/nodejs now?